src/symmetry.jl

```diff
@@ -270,7 +267,7 @@ function symmetrize_forces(model::Model, forces; symmetries)
     symmetrized_forces = zero.(forces)
     for (iel, (element, positions)) in enumerate(atoms)
         for symop in symmetries
-            W, w = get_Ww(symop)
+            W = symop.W; w = symop.w
```

Comment:
Don't like `;`, better:
```suggestion
            W, w = symop.W, symop.w
```